### PR TITLE
Limit payouts

### PR DESCRIPTION
**Pull request by bbenligiray**
Closes https://github.com/api3dao/claims-manager/issues/15
Review after https://github.com/api3dao/claims-manager/pull/17 is merged

### CODE REVIEW

**Review by ErichDylus — APPROVED** [review]

**Comment by bbenligiray** [comment]
@acenolaza I asked for a review again but I'm not sure if it sent you a notification

File: contracts/ClaimsManager.sol [hunk 1]
```diff
@@ -373,14 +386,21 @@ contract ClaimsManager is
...
-        );
+        uint256 settlementAmountInUsd = claimIndexToProposedSettlementAmountInUsd[
+                claimIndex
+            ];
+        uint256 clippedAmountInUsd = updatePolicyCoverage(
+            claim.policyHash,
+            settlementAmountInUsd
+        );
+        uint256 clippedAmountInApi3 = (settlementAmountInApi3 *
+            clippedAmountInUsd) / settlementAmountInUsd;
```
Comment (acenolaza) [inline contracts/ClaimsManager.sol:401]: All this math is in the same unit (wei?), right?

**Review by acenolaza — APPROVED** [review]
Code changes look good  but I have a question.

We are accepting a claim or proposal for X amount in USD but it might happen that in reality we will be accepting less than X if amount exceeds remaining coverage amount. Wouldn't that be misleading? Shouldn't we just prevent creating claims or settlement proposals that would go over the amount in USD?

File: contracts/ClaimsManager.sol (hunk 1 above)
Comment (bbenligiray) [reply contracts/ClaimsManager.sol:401]: Yes, all amounts have 18 decimals

**Pull request merged**